app/Http/Middleware/Backend/AdminLoginMiddleware.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App\Http\Middleware\Backend;
+
+use Closure;
+use App\Model\User;
+use Illuminate\Support\Facades\Auth;
+
+class AdminLoginMiddleware
+{
+    /**
+     * Handle an incoming request.
+     *
+     * @param \Illuminate\Http\Request $request request
+     * @param \Closure                 $next    next
+     *
+     * @return mixed
+     */
+    public function handle($request, Closure $next)
+    {
+        if (Auth::check()) {
+            $user = Auth::user();
+            if ($user->role == User::$role['admin']) {
+                return $next($request);
+            } else {
```

Comment:
- you has return, so you don't need else block.

Reply:
else block into LoginController at login method in try catch